src/Tgstation.Server.Host/ServerFactory.cs

```diff
@@ -49,7 +49,11 @@ public async Task<IServer> CreateServer(string[] args, string updatePath, Cancel
 
 			var basePath = IOManager.ResolvePath();
 			IHostBuilder CreateDefaultBuilder() => Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder(args)
-				.ConfigureAppConfiguration((context, configuration) => configuration.SetBasePath(basePath));
+				.ConfigureAppConfiguration((context, builder) => {
+					builder.AddJsonFile($"appsettings.{context.HostingEnvironment.EnvironmentName}.json", optional: true, reloadOnChange: true)
+					.AddYamlFile($"appsettings.{context.HostingEnvironment.EnvironmentName}.yml", optional: true, reloadOnChange: true); //Allow both appsettings.json and appsettings.yml for backwards compat
```

Comment:
The .json file is already configured by [CreateDefaultBuilder()](https://github.com/dotnet/runtime/blob/6a5a78bec9a6e14b4aa52cd5ac558f6cf5c6a211/src/libraries/Microsoft.Extensions.Hosting/src/Host.cs#L76). We shouldn't need to redo it.

```suggestion
					builder.AddYamlFile($"appsettings.yml", optional: true, reloadOnChange: true)
					.AddYamlFile($"appsettings.{context.HostingEnvironment.EnvironmentName}.yml", optional: true, reloadOnChange: true); //Allow both appsettings.json and appsettings.yml for backwards compat
```